srcalc/src/main/java/gov/va/med/srcalc/domain/variable/DiscreteNumericalVariable.java

```diff
@@ -185,6 +185,11 @@ void setOption(MultiSelectOption option)
             fOption = Objects.requireNonNull(option);
         }
         
+        public boolean isWnl()
```

Comment:
Hibernate is trying to persist this now. Tests don't pass! Did you try running a build?